bundles/org.openhab.binding.cul/src/main/resources/OH-INF/thing/cun_bridge.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="maxcul"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="cun_max_bridge">
```

Comment:
Is `cun` a name or is it an abbreviation of cul network? I would suggest to follow the modbus binding here an name it `tcp`. Assuming it's not a rfc2217 protocol (serial-over-ip), in which case you would not need this bridge as it can be configured using the serial bridge as it also supports configuring it with network address.

Reply:
See response above.